src/plugins/kss.ts

```diff
@@ -57,18 +65,19 @@ export class KssPlugin implements IPlugin<IKssPluginData> {
     }
 }
 
-function convertSection(section: kss.ISection): IKssExample {
+function convertSection(this: ICompiler, section: kss.ISection): IKssExample {
```

Comment:
using map's `this` context in this fashion is pretty hacky. I know it's more performant than using a lambda in the map above, but just one more arg and this doesn't work.

I think i'd prefer to just use an explicit argument and a lambda in the map.